x-pack/plugins/discover_enhanced/public/actions/explore_data/abstract_explore_data_action.ts

```diff
@@ -42,7 +42,12 @@ export abstract class AbstractExploreDataAction<Context extends { embeddable?: I
 
   public async isCompatible({ embeddable }: Context): Promise<boolean> {
     if (!embeddable) return false;
-    if (!this.params.start().plugins.discover.urlGenerator) return false;
+
+    const { core, plugins } = this.params.start();
+    const { capabilities } = core.application;
+
+    if (capabilities.discover) if (!capabilities.discover.show) return false;
```

Comment:
is having 2 `if`s in one row instead of `if(capabilities.discover && !capabilities.discover.show)` indented?

Reply:
Yes

Reply:
I've collapsed them.